### PR TITLE
Chore pipfile

### PR DESCRIPTION
**Pull request by ireznice**

### CODE REVIEW

**Comment by petrSchreiber** [comment]
I think it would be the best to accompany pull request with some context-adding comment. Allows for better async work.

File: web/modeltr/connection.py [hunk 1]
```diff
@@ -35,7 +37,8 @@ def __init__(self, **kwargs):
         for i in range(Settings.app['retries']['db_connection']):
             try:
                 self.async_mode = True if 'async_mode' in kwargs else False
-                self.client = psycopg2.connect(kwargs['dsn'], async_=self.async_mode)
+                temp_async_mode = 1 if self.async_mode else 0
```
Comment (petrSchreiber) [inline web/modeltr/connection.py:40]: Please have a look here:
https://www.geeksforgeeks.org/python-ways-to-convert-boolean-values-to-integer/

**Comment by petrSchreiber** [comment]
PR contains much more than a pip file, is it by accident?
Seems it duplicates most of https://github.com/avast/labmanager-unit-vsphere/pull/123/files

**Review by petrSchreiber — CHANGES_REQUESTED** [review]
Possibly accidental changes in this pull request, much more than pip file.

**Comment by ireznice** [comment]
let's postpone this, I'll rebase the code on previous merged PR

**Pull request closed without merging**